security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogAction.xml

```diff
@@ -24,19 +24,27 @@
         <help>Pre-defined commands for this restart action.</help>
     </field>
     <field>
-        <label>Optional Parameters</label>
+        <label>Required Parameters</label>
         <type>header</type>
+        <style>method_table method_table_configd</style>
     </field>
     <field>
         <id>action.configd</id>
         <label>System Command</label>
         <type>dropdown</type>
         <help>Select a pre-defined system command which should be run for this action.</help>
+        <style>table_optional table_optional_configd</style>
+    </field>
+    <field>
+        <label>Required Parameters</label>
+        <type>header</type>
+        <style>method_table method_table_custom</style>
     </field>
     <field>
         <id>action.custom</id>
```

Comment:
Is this really necessary? this causes quite some security concerns, we've tried to prevent for all other components we've developed.
Creating configd templates is not very complex and makes sure you can only execute safe code. 
If someone adds rm -rf / your firewall is gone....